fe/fe-core/src/main/java/org/apache/doris/qe/DdlExecutor.java

```diff
@@ -563,7 +563,6 @@ private static void checkDdlStmtSupported(DdlStmt ddlStmt) throws DdlException {
                 || ddlStmt instanceof AdminCleanTrashStmt
                 || ddlStmt instanceof AdminRebalanceDiskStmt
                 || ddlStmt instanceof AdminCancelRebalanceDiskStmt
-                || ddlStmt instanceof AlterResourceStmt
```

Comment:
is there any subsequence side effect?

Reply:
> is there any subsequence side effect?

1. i hava readed the source code,  the resource created by `CreateResourceStmt` can be alter, i will not take effect for storage vault

Reply:
`    public void alterResource(AlterResourceStmt stmt) throws DdlException {
        String resourceName = stmt.getResourceName();
        Map<String, String> properties = stmt.getProperties();

        if (!nameToResource.containsKey(resourceName)) {
            throw new DdlException("Resource(" + resourceName + ") dose not exist.");
        }

        Resource resource = nameToResource.get(resourceName);
        resource.modifyProperties(properties);

        // log alter
        Env.getCurrentEnv().getEditLog().logAlterResource(resource);
        LOG.info("Alter resource success. Resource: {}", resource);
    }`